lib/model/store.dart

```diff
@@ -465,6 +465,10 @@ class PerAccountStore extends ChangeNotifier with ChannelStore, MessageStore {
         if (user == null) {
           return; // TODO log
         }
+        // Fields on [User] not updated here:
+        //   userId, dateJoined, isBot, botType, isSystemBot
```

Comment:
What about also making these immutable fields `final`?!

Reply:
Good idea, thanks. Switched to that.